tests/foreman/cli/test_oscap.py

```diff
@@ -269,13 +304,19 @@ def test_positive_create_scap_content_with_valid_originalfile_name(self):
 
         :expectedresults: The scap-content is not created.
```

Comment:
@san7ket is there an error here?

Reply:
@ldjebran my bad, typo :/ , thanks for catching it